src/ducks/components/Sidebar.jsx

```diff
@@ -1,40 +1,54 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import { translate } from 'cozy-ui/react/I18n'
-import { NavLink as RouterLink } from 'react-router-dom'
+import { NavLink as RouterLink, withRouter } from 'react-router-dom'
 import Nav, { NavLink, NavItem, NavIcon, NavText } from 'cozy-ui/react/Nav'
 
 import discoverIcon from 'assets/icons/icon-compass.svg'
 import myAppsIcon from 'assets/icons/icon-cozy-smile.svg'
 
 import { SidebarCategories } from 'ducks/apps/Containers'
 
-export const Sidebar = ({ t }) => (
-  <aside className="o-sidebar">
-    <Nav>
-      <NavItem>
-        <RouterLink
-          to="/discover"
-          className={NavLink.className}
-          activeClassName={NavLink.activeClassName}
-        >
-          <NavIcon icon={discoverIcon} />
-          <NavText>{t('nav.discover')}</NavText>
-        </RouterLink>
-      </NavItem>
-      <SidebarCategories parent="/discover" />
-      <NavItem>
-        <RouterLink
-          to="/myapps"
-          className={NavLink.className}
-          activeClassName={NavLink.activeClassName}
-        >
-          <NavIcon icon={myAppsIcon} />
-          <NavText>{t('nav.myapps')}</NavText>
-        </RouterLink>
-      </NavItem>
-      <SidebarCategories parent="/myapps" />
-    </Nav>
-  </aside>
-)
+import { enabledStoreParts } from 'config'
 
-export default translate()(Sidebar)
+const configMap = {
+  discover: {
+    icon: discoverIcon,
+    labelKey: 'nav.discover'
+  },
+  myapps: {
+    icon: myAppsIcon,
+    labelKey: 'nav.myapps'
+  }
+}
+
+export const Sidebar = React.memo(({ t }) => {
+  const parts = enabledStoreParts || ['discover', 'myapps']
```

Comment:
I think that enabledStoreParts should be passed as props to the <Sidebar /> component, avoiding repetition of  `enabledStoredPats` in to places of our code.

Reply:
How do you see to pass the config using props here?